src/main/java/com/backend/soullive_a/service/impl/ModelServiceImpl.java

```diff
@@ -50,6 +53,35 @@ public ModelResponse getModel(String modelName,Long productId) {
                 .build();
     }
 
+    private void createModelFitness( ProductModel productModel) {
+        String aiComment="ai 한줄 평 준비 중 입니다.";
+        if (productModel.getModel().getId() == 1) {
+            aiComment = "김희애의 우아하고 고급스러운 이미지와 함께 가족적인 이미지가 어우러져 lg 시그니처 브랜드의 프리미엄적인 이미지와 고품질 제품에 잘 어울릴 것이라고 판단 됩니다.";
+        }
```

Comment:
```suggestion
        if (productModel.getModel().getId() == 1) {
            aiComment = "김희애의 우아하고 고급스러운 이미지와 함께 가족적인 이미지가 어우러져 lg 시그니처 브랜드의 프리미엄적인 이미지와 고품질 제품에 잘 어울릴 것이라고 판단 됩니다.";
            scoreUrl="https://soullive-bucket.s3.ap-northeast-2.amazonaws.com/hanjimin_modelFitness_score.png";
        }
```
scoreUrl이랑 aiComment를 합치면 좋을 것 같아 보여요!

Reply:
모델 이미지 키워드, 목표 기업 이미지, 목표 상품 이미지 도 다 넣어 주세요~